roles/download/defaults/main.yml

```diff
@@ -508,7 +520,7 @@ ovn4nfv_k8s_plugin_image_repo: "{{ docker_image_repo }}/integratedcloudnative/ov
 ovn4nfv_k8s_plugin_image_tag: "{{ ovn4nfv_k8s_plugin_image_version }}"
 
 nginx_image_repo: "{{ docker_image_repo }}/library/nginx"
-nginx_image_tag: 1.19
+nginx_image_tag: stable
```

Comment:
Please don't use mutable tags, `stable` is a mutable tag and points to whatever the current version is and makes CI testing irrelevant.

Reply:
I changed it to 1.21